css/style.css

```diff
@@ -1,79 +1,242 @@
+    float: left;
+    padding: 0 10px 10px 0; /* note the padding around a left floated image */
+  }
+
+  /******************************************
+  /* BASE STYLES
+  /*******************************************/
+
+  body {
+       color: #000;
+       font-size: 24px;
+       font-weight: 600;
+       line-height: 1.4;
+       font-family: 'Open Sans', sans-serif;
+       background-color: #f0efef;
+       color: #033048;
+       border:solid #033048 1px;
+       padding:5px;
+       margin: auto;
+       width: 960px;
+  }
+
+
+  header{
+      background-color:#033048;
+      color:#f9e42e;
+      width: 100%;
+      margin: auto;
 }
 
-/* Clear fix hack */
-.clearfix:after {
-     content: ".";
-     display: block;
-     clear: both;
-     visibility: hidden;
-     line-height: 0;
-     height: 0;
+nav{
+  padding-left: 20px;
+  display: inline;
 }
 
-.clear {
-  clear: both;
+  h1{
+    font-size: 40px;
+    color:#f9e42e;
+    padding-left: 50px;
+    display: inline;
+  }
+
+  /*Gutter can be done with just padding*/
+
+  ul{
+    list-style-type: none;
+  }
+
+  ul>li{
+    float: left;
+  }
+
+  li a{
+    font-size: 22px;
+    font-weight: 300;
+    text-decoration: none;
+    display: block;
+    padding: 0 15px;
+    color: #fff;
+    line-height: 1.5;
+  }
+
+ ul>li:last-child{
+  padding-right: 50px;
+  ;
 }
+  .drop{
+    font-size: 80px;
+    font-weight: 800px;
+    padding:0px;
+  }
+
+  .article h3{
+    font-size: 40px;
+    padding-top: 20px;
+    padding-bottom: 20px;
+  }
+
+  .article p{
+    font-weight: 300;
+  }
+
+  .sidebar h4{
+    font-size: 32px;
+    font-weight: 800;
+  }
+
+  .sidebar div{
+    font-weight: 300; 
+  }
+
+  .sidebar p{
+    font-size: 22px;
+    font-weight: 300;
+  }
+
+  .sidebar p+div{
+    font-weight: 600;
+    font-size: 22px;
+  }
+
+  .readmore{
+    text-align: right;
+    padding-top: 25px;
+    padding-bottom: 25px;
+  }
+
+  .signup{
+    background-color: #f9e42e;
+    padding-bottom: 15px;
+    padding-top: 15px;
+    text-align: center;
+  }
+
+  .signup button{
+    font-size: 12px;
+    font-weight: 800;
+    border: 5px;
+    background-color: #033048;
+    color: white;
+    padding: 10px 30px;
+  }
+
+
+  .footer{
+    background-color: #121212;
+    padding-top: 30px;
+    padding-bottom: 30px;
+  }
+
+  .footer div{
+    font-size: 24px;
+    font-weight: 800;
+    color:#f9e42e;
+    text-align: center;
+  }
+
+  .twitter{
+    height: 17px;
+    width: 20px;
+    padding-right: 5px;
+  }
+
+  .facebook{
+    height: 17px;
+    width: 10px;
+    padding-left: 5px;
+  }
+
+  .footer img{
+    height: 17px;
+    width: 20px;
+  }
+
+  .footerimages{
+    margin: auto;
+  }
+
+  .footer footer{
+    font-size: 9px;
+    font-weight: 600;
+  }
+
+  .footer div:last-child{
+    font-size: 10px;
+    color: #ffffff;
+  }
+  /******************************************
+  /* LAYOUT
+  /*******************************************/
+
+  /* Center the container */
+    html{
```

Comment:
I recommend removing this whole selector. `box-sizing` is redundant because its already set up at the top of the file with the `* {}`  selector. Also, in order for the page to go edge to edge like the design, margin and padding will get in your way.